lib/simp/cli/utils.rb

```diff
@@ -51,7 +51,7 @@ def initialize
     end
 
     def get_config
-      return %x{puppet config print}.lines
+      return %x{puppet config print --section=master}.lines
```

Comment:
why is this distinction important?

Reply:
@lnemsick-simp The sections can have different values and this gets the master-specific one and will walk back up to `main` if necessary.